### PR TITLE
Remove softmax from output

### PR DESCRIPTION
**Pull request by rasbt**
I don't think there should be a softmax in the output layer if you use `CrossEntropyLoss` since CrossEntropyLoss takes logits as inputs (and performs a `LogSoftmax` internally): https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html.

### CODE REVIEW

**Comment by Yard1** [comment]
Have you tried running the code?

**Comment by rasbt** [comment]
No, the code generally doesn't work on my computer (getting import errors with the logger). Looking at it carefully, I can't see an issue with the fix. It should run ok.

**Comment by rasbt** [comment]
Fixed it in the example code so that it can be tested by the CI workflow

**Comment by Yard1** [comment]
Thanks!
